packages/@react-aria/utils/src/useResizeObserver.ts

```diff
@@ -24,14 +24,20 @@ export function useResizeObserver<T extends HTMLElement>(options: useResizeObser
         window.removeEventListener('resize', onResize, false);
       };
     } else {
-
       const resizeObserverInstance = new window.ResizeObserver((entries) => {
         if (!entries.length) {
           return;
         }
 
         onResize();
+
+        resizeObserverInstance.unobserve(element);
+
+        requestAnimationFrame(() => {
+          resizeObserverInstance.observe(element);
+        });
```

Comment:
😬 

this seems... bad?

Reply:
Yeah, but apparently this is the way.
The link in the description does a better job of explaining than I can do, especially some 12 months after I last looked into it. The crux of it is
> The solution is to suspend observation and restart on the next animation frame, but only if the element resized while the handler was executing.

Which now that I'm reading that, looks like I missed an if guard checking the original size and new size. I'll update shortly, it's not necessary for today's testing as it is only a warning and there haven't been a ton of requests to deal with it.

Reply:
still, feels like if we are hitting an error here we are probably doing something else wrong? like, this seems like a bandaid?

Reply:
Honestly, it appears to be a browser issue if you read the rest of that post I put in the description.